samples/tests/quickstart_v1beta3_test.py

```diff
@@ -0,0 +1,59 @@
+# # Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from uuid import uuid4
+import pytest
+import sys
+import os
+from google.cloud import storage
+
+from samples import quickstart_sample_v1beta3
+
+
+location = "us"
+project_id = '1012616486416'
+processor_id = '90484cfdedb024f6'
+bucket_name = 'python_docs_samples_test_%s' % uuid4()
+gcs_input_uri = 'gs://cloud-samples-data/documentai/invoice.pdf'
+gcs_output_uri_prefix = uuid4()
+file_name = 'samples/resources/invoice.pdf'
+file_path = os.path.join(os.getcwd(), file_name)
```

Comment:
Very interested in this `os.getcwd()` here - can we not assume that the user will be running the tests from this directory? I'm thinking it might be overkill and that we can make some assumptions, but I also could be totally off base and absolutely welcome further discussion here

Reply:
I think it makes sense to make that assumption, especially based off of similar snippets. I'll adjust.